activation/activation.go

```diff
@@ -223,6 +223,7 @@ func (b *Builder) StartPost(rewardAddress types.Address, dataDir string, space u
 		case InitDone:
 			return fmt.Errorf("already initialized")
 		case InitInProgress:
+			fallthrough
 		default:
```

Comment:
I think that it's better to remove the `default:` clause here, than add a `fallthrough`. There aren't any other values that `b.initStatus` can have that aren't explicitly covered by the `switch` (it won't get to the switch if it's `InitIdle`).

```suggestion
```